src/Compilers/CSharp/Test/Emit3/RefStructInterfacesTests.cs

```diff
@@ -23551,23 +23551,19 @@ public void IsOperator_07()
 class Helper1<T, U>
     where T : allows ref struct
 {
-    static void Test1(U h1)
+    public static void Test1(U h1)
     {
         if (h1 is T)
```

Comment:
Does the spec needs updating? https://github.com/dotnet/csharpstandard/blob/ebf205860866ff7adff0c9876dec8873deadf64c/standard/expressions.md#1212121-the-is-type-operator

> The result of the operation `E is T` [...] is a Boolean value indicating whether `E` is non-null and can successfully be converted to type `T` by a reference conversion, a boxing conversion, an unboxing conversion, a wrapping conversion, or an unwrapping conversion.

This is not one of the listed conversions, right?

Reply:
> This is not one of the listed conversions, right?

It is it not? The check is performed at runtime.